### ISSUE TITLE
Fix entropy and coinformation API

### ISSUE THREAD

**Issue by chebee7i** [dit/dit#70]
Entropy does not accept list of lists for `rvs` or `crvs`. H[X, Y] == H[Z] where Z = (X,Y). So we should have:

```
entropy(d, [[0,1]]) == entropy(d, [[0], [1]]) == entropy(d, [0,1])
```

where the last form a convenience for users. The same holds for `crvs` in `coinformation`: it does not yet accept nested lists.

**Comment by Autoplectic** [comment]
fixed in a3dec54fa25f9aadb3a2f7e315ab11a2a0795a8d

**Comment by chebee7i** [comment]
Probably good, but I'll double check this later. There is one place in the code (in `coalesce`) where the distinction between a flattened version and a non-flatten version matters.
